tests/rbd/rbd_migration_2diff_ec_pool_ec.py

```diff
@@ -0,0 +1,157 @@
+"""Module to execute Live migration feature test case.
+Pre-requisites :
+We need cluster configured with atleast one client node with ceph-common package,
+conf and keyring files
+Test cases covered -
+1) CEPH-83573322 - Migrate the images from 2 different source pools to destination
+and check the behaviour
+Test Case Flow -
+1. Create two source EC pool with different k_m values and an image
+2. Create an EC pool for destination pool
+3. Mount the images and run io from clients
+4. Stop IO and prepare migration process from 2 source pools to destination pool
+5. Start IO from client on destination images
+6. Execute migration
+7. Commit migration and check the destination image info
+"""
+
+from ceph.parallel import parallel
+from tests.rbd.exceptions import RbdBaseException
+from tests.rbd.rbd_utils import (
+    Rbd,
+    initial_rbd_config,
+    verify_migration_commit,
+    verify_migration_state,
+)
+from utility.log import Log
+from utility.utils import run_fio
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """verify live migration of images from 2 different source pools to destination pool.
+    Args:
+        kw: Key/value pairs of configuration information to be used in the test
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+    """
+    log.info(
+        "Starting execution of live migration from 2 source EC pool to destination EC pool"
+    )
+    rbd = Rbd(**kw)
+    config = {
+        "ec-pool-only": True,
+        "ec-pool-k-m": "2,1",
+        "ec_pool_config": {
+            "pool": "rbd_ECpool1",
+            "image": "rbd_image_ec1",
+            "data_pool": "data_pool_ec1",
+            "size": "10G",
+        },
+    }
+    kw["config"].update(config)
+    initial_rbd_config(**kw)
+    image1 = f'{config["ec_pool_config"]["image"]}'
+    ecpool1 = f'{config["ec_pool_config"]["pool"]}'
+    datapool1 = f'{config["ec_pool_config"]["data_pool"]}'
+    src_spec1 = ecpool1 + "/" + image1
```

Comment:
Shall we please not use `ecpool` variable name for pools for which pools are not ec ?

Reply:
Done